pyproject.toml

```diff
@@ -41,6 +41,8 @@ dependencies = [
   "pytz>=2023.3",
   "pyyaml>=6.0",
   "tomli>=2.0.1",
+  "pandas>=1.5.0",
+  "scikit-learn>=1.2.0"
```

Comment:
I don't want to make these to level dependencies + we already have pandas as an extra dependency.
Let's make an extra called "ml" and put scikit-learn in there. That way you can install the extra dependency as `koheesio[pandas,ml]`